app/src/ui/publish-repository/publish-repository.tsx

```diff
@@ -10,18 +10,22 @@ import { caseInsensitiveCompare } from '../../lib/compare'
 import { sanitizedRepositoryName } from '../add-repository/sanitized-repository-name'
 import { Octicon, OcticonSymbol } from '../octicons'
 
-interface IPublishRepositoryProps {
-  /** The user to use for publishing. */
-  readonly account: Account
+export interface IGHEPublicationSettings {
```

Comment:
:art: can we avoid the `GHE` abbreviation in favour of `Enterprise` for the type name and `enterprise` for the `kind` value?